noetic/distribution.yaml

```diff
@@ -1731,7 +1731,7 @@ repositories:
       tags:
         release: release/noetic/{package}/{version}
       url: https://github.com/shadow-robot/ethercat_grant-release.git
-      version: 0.2.5-3
+      version: 0.2.5-2
```

Comment:
This seems very wrong; the numbers are going backwards.  What are you trying to accomplish here?